binary/expression.proto

```diff
@@ -251,3 +376,5 @@ message AggregateFunction {
   AggregationPhase phase = 4;
   Type output_type = 5;
 }
+
+
```

Comment:
Kill these newlines

Reply:
I'm going to keep adding these just to motivate someone to add more checks :D